src/Common/src/System/Security/Cryptography/Asn1/GeneralNameAsn.cs

```diff
@@ -3,16 +3,15 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Runtime.InteropServices;
-using System.Security.Cryptography.Asn1;
 
-namespace System.Security.Cryptography.Pkcs.Asn1
+namespace System.Security.Cryptography.Asn1
 {
     [Choice]
     [StructLayout(LayoutKind.Sequential)]
-    internal struct GeneralName
+    internal struct GeneralNameAsn
     {
         [ExpectedTag(0, ExplicitTag = true)]
-        internal OtherName? OtherName;
```

Comment:
According to the test values from `SubjectAlternativeNameBuilderTests` these tags should not be EXPLICIT. Will investigate.